website/api/controllers/webhooks/receive-from-github.js

```diff
@@ -616,8 +616,10 @@ module.exports = {
 
       // Only continue if this release came from the fleetdm/fleet repo,
       if(owner === 'fleetdm' && repo === 'fleet') {
-        // Only send requests for releases with tag names that start with 'fleet'
-        if(release && _.startsWith(release.tag_name, 'fleet-v')) {
+        if(release
+          && _.startsWith(release.tag_name, 'fleet-v')// Only send requests for releases with tag names that start with 'fleet'
+          && _.endsWith(release.tag_name, '.0')// Only send requests if the release is a major or minor versions.
```

Comment:
```suggestion
          && _.endsWith(release.tag_name, '.0')// Only send requests if the release is a major or minor version. This works because all Fleet semvers have 2 periods. 
```